src/views/ResourceSavingScene.tsx

```diff
@@ -45,7 +45,6 @@ export default class ResourceSavingScene extends React.Component<Props> {
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    overflow: 'hidden',
```

Comment:
This will disable the `removeClippedSubviews` optimization that we apply to optimize memory usage.